lib/ecto_enum/postgres/use.ex

```diff
@@ -77,10 +79,18 @@ defmodule EctoEnum.Postgres.Use do
         def create_type() do
           Ecto.Migration.execute(unquote(create_sql), unquote(drop_sql))
         end
+
+        def alter_type() do
```

Comment:
Hi @sofakingworld. Can we rename the function to `add_value`? `alter_type` can be a bunch of different things like renaming the type.